### ISSUE TITLE
equations

### ISSUE THREAD

**Issue by xrubio** [ercrema/CultureAllee#6]
I adedd some of the equations to the LaTeX manuscript here: fdd1ffa1ecb49046bee5ba4a46f2052542c8cfa2
We have to confirm that these are the final equations, and add the rest.

**Comment by ercrema** [comment]
Did you use the ones contained here: fdd1ffa1ecb49046bee5ba4a46f2052542c8cfa2

**Comment by xrubio** [comment]
Yep...this is my commit :-D

**Comment by ercrema** [comment]
No I mean the one here: https://github.com/ercrema/CultureAllee/blob/fdd1ffa1ecb49046bee5ba4a46f2052542c8cfa2/manuscript/equations/equations.tex

**Comment by xrubio** [comment]
shit...I didn't see the file! In any case my version uses 'align', which is better (I think)...
Anyway I'll work these days in the manuscript according to the structure in the wiki.

**Comment by ercrema** [comment]
Yes, the only thing we need to decide for the equations is whether we want to use the simplified version for the competition model or not.
